docs/gds-supported-platform/per-namespace-gateway-opt-in.md

```diff
@@ -0,0 +1,49 @@
+# Per-namespace istio gateways
+
+We are migrating from using a single `ingressgateway` in `istio-system` to an `ingressgateway` in each namespace. At present this requires an "opt-in" via `-cluster-config` values settings to enable.
```

Comment:
Recommend you make this more active. E.g. "You must opt into / enable an `ingressgateway` in each namespace."

Reply:
Updated.